tools/cockpit.spec

```diff
@@ -499,6 +514,34 @@ test -f %{_bindir}/firewall-cmd && firewall-cmd --reload --quiet || true
 %postun ws
 %systemd_postun_with_restart cockpit.socket cockpit.service
 
+%package selinux-policy
+Summary: SELinux policies
+Requires: cockpit-ws = %{version}-%{release}
+%{?selinux_requires}
+
+%description selinux-policy
+Custom SELinux policy module for cockpit-ws.
+
+%files selinux-policy
+%{_datadir}/selinux/packages/%{selinuxtype}/%{name}.pp.bz2
+%{_mandir}/man8/%{name}_session_selinux.8.*
+%{_mandir}/man8/%{name}_ws_selinux.8.*
+%ghost %{_sharedstatedir}/selinux/%{selinuxtype}/active/modules/200/%{name}
+
+%pre selinux-policy
+%selinux_relabel_pre -s %{selinuxtype}
+
+%post selinux-policy
+%selinux_modules_install -s %{selinuxtype} %{_datadir}/selinux/packages/%{selinuxtype}/%{name}.pp.bz2
+
+%postun selinux-policy
+if [ $1 -eq 0 ]; then
+    %selinux_modules_uninstall -s %{selinuxtype} %{name}
+fi
+
+%posttrans selinux-policy
+%selinux_relabel_post -s %{selinuxtype}
```

Comment:
Since we're dealing with a service, we should move selinux_relabel_post to %post/%postun and add service restart after relabeling... see: https://fedoraproject.org/wiki/SELinux/IndependentPolicy#Example_spec_file_changes_to_incorporate_-selinux_subpackage (version for daemons)